geopyspark/tests/base_test_class.py

```diff
@@ -4,4 +4,4 @@
 
 
 class BaseTestClass(unittest.TestCase):
-    geopysc = GeoPyContext(master="local[*]", appName="test")
+    geopysc = GeoPyContext(master="local[1]", appName="test")
```

Comment:
Is this behavior only seen on Travis? If so, then we could just have a condition where it creates a `GeoPyContext` based its environment.